src/containers/products/ResearchHubProductPage.js

```diff
@@ -1,36 +1,151 @@
-/*
- * @flow
- */
-
+// @flow
 import React from 'react';
 import styled from 'styled-components';
 
 import ContactUsSection from '../contactus/ContactUsSection';
-import FooterSection from '../footer/FooterSectionWithPageLinks';
+import Feature1 from '../../assets/productfeatureicons/researchhub/feature1.svg';
+import Feature2 from '../../assets/productfeatureicons/researchhub/feature2.svg';
+import Feature3 from '../../assets/productfeatureicons/researchhub/feature3.svg';
+import FeatureImg1 from '../../assets/images/permissions-feature-shot.png';
+import FeatureImg2 from '../../assets/images/researchhub/research-hub-feature-shot-2.png';
+import FooterSectionWithPageLinks from '../footer/FooterSectionWithPageLinks';
+import PlaceholderIcon from '../../assets/logos/placeholder-small-logo.svg';
 import PageSection from '../../components/layout/NewPageSection';
+import ProductImg from '../../assets/images/researchhub/research-hub-product-shot.png';
 import ScrollToTopOnMount from '../../components/layout/ScrollToTopOnMount';
 
 import { PageIntro } from '../../components/headers/PageHeaders';
-import { Content } from '../../components/layout/IntroComponents';
+import { Content, PageDescriptionWrapper } from '../../components/layout/IntroComponents';
+import {
+  ExtendedFeatureDescription,
+  ExtendedFeatureHeader,
+  ExtendedFeaturesContent,
+  FeatureOverviewDescription,
+  FeatureOverviewTitle,
+  FeatureOverviewWrapper,
+  FeatureShot,
+  Icon,
+  ProductFeaturesContent,
+  ProductOverviewDescriptionWrapper,
+  ProductOverviewWrapper,
+  ProductShot,
+  ProductStamp,
+} from './styled/StyledProductComponents';
 import { NEUTRALS } from '../../core/style/Colors';
-import { MEDIA_QUERY_LG } from '../../core/style/Sizes';
+import { MEDIA_QUERY_LG, MEDIA_QUERY_MD, MEDIA_QUERY_TECH_SM } from '../../core/style/Sizes';
 
-const ComingSoonContent = styled(Content)`
+const IntroContent = styled(Content)`
   @media only screen and (min-width: ${MEDIA_QUERY_LG}px) {
-    margin: 300px auto;
+    margin: 70px auto 450px auto;
+  }
+`;
+
+const ResearchHubIntro = styled(PageIntro)`
+  margin-bottom: 32px;
+  width: 720px;
+
+  @media only screen and (max-width: ${MEDIA_QUERY_TECH_SM}px) {
+    width: 100%;
+  }
+`;
+
+const ResearchHubDescription = styled(PageDescriptionWrapper)`
+  margin: 0;
+`;
+
+const ResearchHubProductShot = styled(ProductShot)`
+  @media only screen and (max-width: ${MEDIA_QUERY_MD}px) {
+    top: 600px;
+  }
+
+  @media only screen and (max-width: ${MEDIA_QUERY_TECH_SM}px) {
+    top: 690px;
+  }
+`;
+
+const PermissionsFeatureShot = styled(FeatureShot)`
+  height: 315px;
+
+  @media only screen and (max-width: ${MEDIA_QUERY_TECH_SM}px) {
+    height: 197px;
```

Comment:
These are some very interesting choices. Have you considered using relative units instead of absolute (eg. vh vs px)?